cmake/IgnPackaging.cmake

```diff
@@ -168,8 +174,8 @@ endmacro()
 # If the target is a component, pass in the COMPONENT argument followed by the
 # component's name.
 #
-# NOTE: This will be called automatically by ign_create_core_library(~) and
-#       ign_add_component(~), so users of ignition-cmake should not call this
+# NOTE: This will be called automatically by gz_create_core_library(~) and
+#       gz_add_component(~), so users of ignition-cmake should not call this
```

Comment:
this change is out of sequence, but it's not a big deal since it will change soon anyway